ui/src/app/machines/views/MachineList/MachineList.js

```diff
@@ -458,10 +465,6 @@ const MachineList = ({ selectedMachines = [], setSelectedMachines }) => {
     dispatch(zoneActions.fetch());
   }, [dispatch, machinesLoaded]);
 
-  useEffect(() => {
```

Comment:
This is replaced with `useMemo` above. It seemed that this was only in state to prevent unnecessary calls, but as this was updating state it meant that any state updates to machines was causing an infinite loop. Replaced with useMemo as it is probably the correct way of preventing reruns.